src/main/resources/org/jvnet/hudson/plugins/collapsingconsolesections/CollapsingSectionNote/script.js

```diff
@@ -1,22 +1,26 @@
-function doToggle(o)
-{
+// The doToggle method is used by CollapsingSectionAnnotator.java
+// eslint-disable-next-line no-unused-vars
+function doToggle(o) {
     var section = o.parentNode.parentNode;
     if (section.nextElementSibling) {
         if (section.nextElementSibling.className === "collapsed") {
             section.nextElementSibling.className = "expanded";
             o.innerHTML = "Hide Details";
-        } else {
+        }
+        else {
```

Comment:
this is unusual formatting for javascript, else would normally be on the same line as the `}`, what about setting up `prettier`?

Reply:
Taht is because I went to use `customize()` https://github.com/eslint-stylistic/eslint-stylistic/blob/main/packages/eslint-plugin/configs/customize.ts  which turns out to have non default such as `braceStyle = 'stroustrup'`.   I will adjust the ESLint config to list rules explicitly instead.

Reply:
I went to pass `braceStyle "1tbs"` to the `customize()` factory. I don't know why the factory has a different default than the rule.